_posts/2024-04-20-16.0.0-release.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Apache Arrow 16.0.0 Release"
+date: "2024-04-20 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 16.0.0 release. This covers
+over 3 months of development work and includes [**385 resolved issues**][1]
+on [**586 distinct commits**][2] from [**119 distinct contributors**][2].
+See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 15.0.0 release, Jeffrey Vo, Jay Zhan, Bryce Mecum and Sarah Gilmore
+have been invited to be committers.
+No new members have joined the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## C Data Interface notes
+
+
+## Arrow Flight RPC notes
+
+
+## C++ notes
+
+
+### Parquet
+
+### Miscellaneous
+
+## C# notes
+
+## Go Notes
```

Comment:
```suggestion
## Go Notes

* The Golang Arrow and Parquet libraries now require Go 1.21+ ([GH-40733](https://github.com/apache/arrow/issues/40733)) 

### Bug Fixes

#### Arrow

* FlightSQL Driver will now properly handle concurrent result sets instead of pulling the entire result into memory ([GH-40089](https://github.com/apache/arrow/issues/40089))
* FlightSQL driver will now correctly respect the `DriverConfig.TLSEnabled` field ([GH-40097](https://github.com/apache/arrow/issues/40097))
* Fixed a panic on 32-bit architectures ([GH-40672](https://github.com/apache/arrow/issues/40672))
* Corrected a precision loss for Decimal types when converting to JSON ([GH-40693](https://github.com/apache/arrow/issues/40693))
* Fixed an issue with `array.RecordBuilder` when using a NullType column ([GH-40719](https://github.com/apache/arrow/issues/40719))

#### Parquet

* Fixed panic when writing a DeltaBinaryPacked column containing only nulls ([GH-35718](https://github.com/apache/arrow/issues/35718))
* Fixed a panic when writing a ListOf(DeltaBinaryPacked) field with no data ([GH-39309](https://github.com/apache/arrow/issues/39309))
* Arrow DATE64 types will now be properly coerced into Parquet DATE[32-bit] logical type ([GH-39456](https://github.com/apache/arrow/issues/39456))
* Fixed the timezone semantics for timestamp conversion from Arrow to Parquet ([GH-39466](https://github.com/apache/arrow/issues/39466))
* Corrected an inaccuracy with `RowGroupTotalCompressedBytes` and `RowGroupTotalBytesWritten` for Parquet file writer ([GH-39870](https://github.com/apache/arrow/issues/39870))
* Fixed the `TotalCompressedBytes` count when falling back to plain encoding if a dictionary is too large ([GH-39921](https://github.com/apache/arrow/issues/39921))
* Fixed a bug when reslicing a nullable dictionary in the chunked writer ([GH-39925](https://github.com/apache/arrow/issues/39925))

### Enhancements

#### Arrow

* Users can now access the underlying `MemoTable` of a dictionary builder ([GH-38988](https://github.com/apache/arrow/issues/38988))
* Added an option to provide a string replacer for CSV writing ([GH-39552](https://github.com/apache/arrow/issues/39552))
* Flight: Cookies can be copied to another connection to reuse existing credentials ([GH-39837](https://github.com/apache/arrow/issues/39837))
* Flight: enable PollFlightInfo for Flight SQL clients/servers ([GH-39574](https://github.com/apache/arrow/issues/39574))
* Added the ability to create a PreparedStatement from persisted data and provided access for FlightSQL users to the PreparedStatement handle property ([GH-39774](https://github.com/apache/arrow/issues/39774) [GH-39910](https://github.com/apache/arrow/issues/39910))
* FlightRPC Session management extensions have been implemented ([GH-40155](https://github.com/apache/arrow/issues/40155))

#### Parquet

* Can now register new compression codecs for Parquet ([GH-40113](https://github.com/apache/arrow/issues/40113))
* Parquet footers can be incrementally written without closing the file ([GH-40630](https://github.com/apache/arrow/issues/40630))

```